versioned_docs/version-1.4/installation/installation.md

```diff
@@ -0,0 +1,122 @@
+---
+sidebar_position: 1
+---
+
+# Installation
+
+## Prerequisites
+
+- Make sure you have installed the latest version of [`Neovim v0.9.0+`](https://github.com/neovim/neovim/releases/latest).
+- Have [`git`](https://cli.github.com/), [`make`](https://www.gnu.org/software/make/), [`pip`](https://pypi.org/project/pip/), [`python`](https://www.python.org/), [`npm`](https://npmjs.com/), [`node`](https://nodejs.org/), [`cargo`](https://www.rust-lang.org/tools/install) and [`ripgrep`](https://github.com/BurntSushi/ripgrep) installed on your system.
+- [Resolve `EACCES` permissions when installing packages globally](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally) to avoid error when installing packages with npm.
+- [`PowerShell 7+`](https://learn.microsoft.com/en-us/powershell/scripting/whats-new/migrating-from-windows-powershell-51-to-powershell-7?view=powershell-7.2) (for Windows)
+
+## Optional Feature Prerequisites
+- Install [`lazygit`](https://github.com/jesseduffield/lazygit#installation). This enables `<leader>gg` to launch `lazygit` for intergrated and enhanced Git experience while in `lvim`.
+
+## Release
+
+(Neovim 0.9.5)
+
+No alarms and No surprises:
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="linux/macos" label="Linux/MacOs">
+
+```bash
+LV_BRANCH='release-1.4/neovim-0.9' bash <(curl -s https://raw.githubusercontent.com/LunarVim/LunarVim/release-1.3/neovim-0.9/utils/installer/install.sh)
+```
+
+</TabItem>
+<TabItem value="windows" label="Windows">
+
+```powershell
+pwsh -c "`$LV_BRANCH='release-1.4/neovim-0.9'; iwr https://raw.githubusercontent.com/LunarVim/LunarVim/release-1.3/neovim-0.9/utils/installer/install.ps1 -UseBasicParsing | iex"
```

Comment:
`https://raw.githubusercontent.com/LunarVim/LunarVim/release-1.3/` should be `release-1.4`

Reply:
same for Linux/MacOs

Reply:
@vonPB, not sure if any of the changes in 51f56ac or 6f2a875 are needed, I would have thought current would just be symlinks instead of a complete separate copy, but if it's meant to be for master then we were specifying the wrong branch anyway.

Reply:
Symlinks would probably make a lot of sense u are right

Reply:
> Symlinks would probably make a lot of sense u are right

it's a bit convoluted how we're using these files because it should've been a different branch or something, from the PR template:

> [IMPORTANT]
> Our docs are versioned:
> - files in `/docs` are for the next version, you most likely want to edit files in this folder
> - files in `/versioned-docs` are frozed docs for current and older versions, edits here won't be included in the next version

I've updated `current` to now point correctly to master, even in i18n.

Reply:
@vonPB, let's merge this for now and apply further fixes later